src/z3c/rml/attr.py

```diff
@@ -334,18 +337,25 @@ def fromUnicode(self, value):
                     'The package-path-pair you specified was incorrect. %s' %(
                     getFileInfo(self.context)))
             modulepath, path = result.groups()
-            module = __import__(modulepath, {}, {}, (modulepath))
-            value = os.path.join(os.path.dirname(module.__file__), path)
-        # If there is a drive name in the path, then we want a local file to
-        # be opened. This is only interesting for Windows of course.
-        if os.path.splitdrive(value)[0]:
-            value = 'file:///' + value
+            module = import_module(modulepath)
+            if six.PY2:
+                value = os.path.join(os.path.dirname(module.__file__), path)
+            else:
+                # PEP 420 namespace support means that a module can have
+                # multiple paths
+                for module_path in module.__path__:
+                    value = os.path.join(module_path, path)
+                    if os.path.exists(value):
+                        break
+        # Under Python 3 all platforms need a protocol for local files
+        if not six.moves.urllib.parse.urlparse(value).scheme:
+            value = 'file:///' + os.path.abspath(value)
```

Comment:
For some reason the file log handler requires four slashes. If you change this line to just "file://" then the test for tag-log.rml will fail in all versions of Python.